src/main/webapp/static/css/styles.css

```
@@ -207,3 +207,14 @@ form {
 #formLoadingOverlay > .preloader-wrapper {
     top: 50%;
 }
+
+.donations-list {
+    margin-bottom: 20px;
```

Use relative margin instead of fixed margin:

```suggestion
    margin-bottom: 1rem;
```